jans-auth-server/server/src/main/java/io/jans/as/server/register/ws/rs/RegisterValidator.java

```diff
@@ -209,7 +209,17 @@ private void validateRequestObjectHmac(HttpServletRequest httpRequest, SsaValida
         log.trace("Request object validation result: {}", validSignature);
         if (validSignature) {
             return;
+        } else {
+            String url = httpRequest.getRequestURL().toString();
+            List<String> disallowedUrls = appConfiguration.getBlockedUrls();
+            for (String disallowedUrl : disallowedUrls) {
+                if (url.contains(disallowedUrl)) {
+                    log.error("URL '{}' is disallowed.", url);
+                    throw new InvalidJwtException("The request object contains a disallowed URL: " + url);
+                }
+            }
```

Comment:
I'm not sure I understand what are we trying to do here. If RP makes request with request object which is hashed then if that hash is invalid we check request url and throw some misleading exception instead of let caller know that hash is wrong.

Reply:
@433B I wonder why PR is changing jans-auth-server if ticket is asking about changes in webhook inside config-api ?

Reply:
@yuriyz may I ask, which config-api are you talking about?

Reply:
Please check this comment https://github.com/JanssenProject/jans/issues/8574#issuecomment-2467728401
There is link to exact resource class.